cores/cv32e40p/user_manual/source/pulp_hw_loop.rst

```diff
@@ -37,27 +37,39 @@ mapped into the CSR address space and thus it is possible to read and
 write them via csrr and csrw instructions. Since hardware loop registers
 could be overwritten in when processing interrupts, the registers have
 to be saved in the interrupt routine together with the general purpose
-registers.
-
-CSR Mapping
------------
-
-+-------------------+-----------+------------+------------+-------------------+--------------+-------+---------------------------+
-|  CSR Address                                            |                   |              |       |                           |
-+-------------------+-----------+------------+------------+                   |              |       |                           |
-|   11:10           |   9:8     |   7:6      |   5:0      |  Hex              | Name         | Acc.  | Decription                |
-+===================+===========+============+============+===================+==============+=======+===========================+
-| 01                | 11        | 10         | 110000     | 0x7C0             | lpstart[0]   | R/W   | Hardware Loop 0 Start     |
-+-------------------+-----------+------------+------------+-------------------+--------------+-------+---------------------------+
-| 01                | 11        | 10         | 110001     | 0x7C1             | lpendt[0]    | R/W   | Hardware Loop 0 End       |
-+-------------------+-----------+------------+------------+-------------------+--------------+-------+---------------------------+
-| 01                | 11        | 10         | 110010     | 0x7C2             | lpcount[0]   | R/W   | Hardware Loop 0 Counter   |
-+-------------------+-----------+------------+------------+-------------------+--------------+-------+---------------------------+
-| 01                | 11        | 10         | 110100     | 0x7C4             | lpstart[1]   | R/W   | Hardware Loop 0 Start     |
-+-------------------+-----------+------------+------------+-------------------+--------------+-------+---------------------------+
-| 01                | 11        | 10         | 110101     | 0x7C5             | lpend[1]     | R/W   | Hardware Loop 1 End       |
-+-------------------+-----------+------------+------------+-------------------+--------------+-------+---------------------------+
-| 01                | 11        | 10         | 110110     | 0x7C6             | lpcount[1]   | R/W   | Hardware Loop 1 Counter   |
-+-------------------+-----------+------------+------------+-------------------+--------------+-------+---------------------------+
-
-Table 6: Hardware-Loop CSR Mapping
+registers. The CS HWLoop registers are described in the control_status_registers
```

Comment:
Can you add a remark at the start of this chapter that  the hardware loop functionality is only included if the  ``PULP_HWLP`` parameter is 1?